code/game/objects/items/weapons/RCL.dm

```diff
@@ -0,0 +1,152 @@
+/obj/item/weapon/twohanded/rcl
+	name = "rapid cable layer	"
+	desc = "A device used to rapidly deploy cables. It has screws on the side which can be removed to slide off the cables. Do not use without insulation!"
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "rcl-0"
+	item_state = "rcl-0"
+	opacity = FALSE
+	force = 5 //Plastic is soft
+	throwforce = 5
+	throw_speed = 1
+	throw_range = 7
+	w_class = WEIGHT_CLASS_NORMAL
+	origin_tech = "engineering=4;materials=2"
+	var/max_amount = 90
+	var/active = FALSE
+	var/obj/structure/cable/last = null
+	var/obj/item/stack/cable_coil/loaded = null
```

Comment:
@Fox-McCloud the reason for removing the ` = null`s is mainly for readability and anti-redundancy. e.g. it's quicker to see that a declared variable has no `= anything` therefore it's null. Otherwise, you'd have to parse that it IS being assigned something only to have to realize it's still null